src/backup/blog.tsx

```diff
@@ -96,13 +86,14 @@ const CustomPage = (props: PageProps) => {
                       <h3>{post.frontmatter?.title}</h3>
                       <p>{post.frontmatter?.description}</p>
                     </TextElement>
-                    {post.frontmatter?.featuredImage && post.frontmatter?.featuredImage?.childImageSharp && (
+                    {post.frontmatter?.featuredImage && (
                       <ImageContainer>
-                        <ResponsiveImage
-                          image={{
-                            altText: post.frontmatter?.title ?? '',
-                            src: post.frontmatter?.featuredImage
-                          }}
+                        <img
+                          src={`/src/content/blog/${post.fields?.slug?.split('/').pop()}/${
+                            post.frontmatter.featuredImage
+                          }`}
+                          alt={post.frontmatter?.title ?? ''}
+                          style={{ width: '100%', height: '200px', objectFit: 'cover' }}
```

Comment:
This negates the `srcset` and `sizes` features of the optimized responsive images that are generated using the `ResponsiveImage` component. Before merging this change I would do a before and after comparison on the size of the images served.

Reply:
I just noticed that this is a backup file, it is not used in our build. I wonder why it had to be updated!

Reply:
It apparently made the gatsby build fail.